 - Kill the wineboot.c update windows code
 
 It's rather annoying and serves no purpose. I actually don't get why this
 is even implemented in Wine? If i've updated wine packages, then I'm aware that
 I'm using an updated build... If i create a prefix - I know that's what I'm 
 doing, having a window pop up is kind of pointless.


--- a/programs/wineboot/wineboot.c.orig	2020-07-20 22:14:50.321809270 -0400
+++ b/programs/wineboot/wineboot.c	2020-07-20 22:13:53.143769493 -0400
@@ -1281,37 +1281,6 @@ static BOOL start_services_process(void)
     return TRUE;
 }
 
-static INT_PTR CALLBACK wait_dlgproc( HWND hwnd, UINT msg, WPARAM wp, LPARAM lp )
-{
-    switch (msg)
-    {
-    case WM_INITDIALOG:
-        {
-            DWORD len;
-            WCHAR *buffer, text[1024];
-            const WCHAR *name = (WCHAR *)lp;
-            HICON icon = LoadImageW( 0, (LPCWSTR)IDI_WINLOGO, IMAGE_ICON, 48, 48, LR_SHARED );
-            SendDlgItemMessageW( hwnd, IDC_WAITICON, STM_SETICON, (WPARAM)icon, 0 );
-            SendDlgItemMessageW( hwnd, IDC_WAITTEXT, WM_GETTEXT, 1024, (LPARAM)text );
-            len = lstrlenW(text) + lstrlenW(name) + 1;
-            buffer = HeapAlloc( GetProcessHeap(), 0, len * sizeof(WCHAR) );
-            swprintf( buffer, len, text, name );
-            SendDlgItemMessageW( hwnd, IDC_WAITTEXT, WM_SETTEXT, 0, (LPARAM)buffer );
-            HeapFree( GetProcessHeap(), 0, buffer );
-        }
-        break;
-    }
-    return 0;
-}
-
-static HWND show_wait_window(void)
-{
-    HWND hwnd = CreateDialogParamW( GetModuleHandleW(0), MAKEINTRESOURCEW(IDD_WAITDLG), 0,
-                                    wait_dlgproc, (LPARAM)prettyprint_configdir() );
-    ShowWindow( hwnd, SW_SHOWNORMAL );
-    return hwnd;
-}
-
 static HANDLE start_rundll32( const WCHAR *inf_path, BOOL wow64 )
 {
     WCHAR app[MAX_PATH + ARRAY_SIZE(L"\\rundll32.exe" )];
@@ -1467,7 +1436,6 @@ static void update_wineprefix( BOOL forc
 
         if ((process = start_rundll32( inf_path, FALSE )))
         {
-            HWND hwnd = show_wait_window();
             for (;;)
             {
                 MSG msg;
@@ -1479,8 +1447,7 @@ static void update_wineprefix( BOOL forc
                 }
                 else while (PeekMessageW( &msg, 0, 0, 0, PM_REMOVE )) DispatchMessageW( &msg );
             }
-            DestroyWindow( hwnd );
-        }
+        }
         install_root_pnp_devices();
         update_user_profile();
         create_etc_stub_files();
